components/header.js

```diff
@@ -0,0 +1,26 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <Title>CommuniFI</Title>
+    </StyledHeader>
+  );
+}
+
+const StyledHeader = styled.div`
```

Comment:
Same here. Maybe styling just the `Title` might be enough.

Reply:
Thats true but i know that I'm about to add a subtitle to fix #7 . So i need general styling